updates/13.sql

```diff
@@ -0,0 +1,4 @@
+SET SESSION sql_mode='NO_AUTO_VALUE_ON_ZERO';
+ALTER TABLE einzelbeleg DROP FOREIGN KEY einzelbeleg_QuelleGattungID;
+ALTER TABLE selektion_quellengattung MODIFY COLUMN ID INT NOT NULL auto_increment;
+ALTER TABLE einzelbeleg ADD CONSTRAINT einzelbeleg_QuelleGattungID FOREIGN KEY (QuelleGattungID) REFERENCES selektion_quellengattung(ID) ON UPDATE CASCADE;
```

Comment:
I think the foreign key must be restored before the auto increment is set else the references will break during migration

Reply:
I have changed the order